mail_activity_partner/tests/test_mail_activity_partner.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2018 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+
+class TestMailActivityPartner(TransactionCase):
+    def setUp(self):
```

Comment:
@AdriaGForgeFlow sorry I didn't check tests before: seems you didn't see my talk on tests best practices... :stuck_out_tongue: 

Can you:
1. use SavepointCase
2. disable tracking
3. disable reset password email
?

eg:

```py
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        # disable tracking test suite wise
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
        cls.user_model = cls.env['res.users'].with_context(no_reset_password=True)
```

Reply:
Hi @simahawk, actually I did watch the talk and found it very interesting :clap: 

The only issue is that I had forgotten about this PR at all :sweat_smile: 
I'm going to check now, thanks for the review!

Reply:
Done, can you check now? @simahawk